lib/haml_lint/options.rb

```diff
@@ -21,7 +21,7 @@ def parse(args)
       end.parse!(args)
 
       # Any remaining arguments are assumed to be files
-      @options[:files] = args
+      @options[:files] = args.empty? ? ["."] : args
```

Comment:
This changes the behavior of the CLI tool. Is it worth adding something to the README about this?